uber/templates/regform.html

```diff
@@ -429,20 +445,17 @@ <h4 class="h4-inline">
         </div>
     </div>
     <div class="form-group address_details">
-        <label for="region" class="col-sm-2 control-label">State</label>
-        <div class="col-sm-3">
-            <input type="text" name="region" class="form-control" placeholder="State/Province/Region" value="{{ attendee.region }}" data-geo="administrative_area_level_1" />
-        </div>
-
-        <label for="zip_code" class="col-sm-1 control-label">ZIP</label>
-        <div class="col-sm-2">
-            <input type="text" name="zip_code" class="form-control" placeholder="ZIP/PostCode" value="{{ attendee.zip_code }}" data-geo="postal_code" />
+        <label for="region" class="col-sm-2 control-label">State/Region</label>
+        <div class="col-sm-6">
+            <select name="region" class="form-control" placeholder="State/Province" autofocus="autofocus" autocorrect="off" autocomplete="off" data-geo="administrative_area_level_1">
+                {% include "region_opts.html" %}
+            </select>
         </div>
     </div>
     <div class="form-group address_details">
-        <label for="country" class="col-sm-2 control-label">Country</label>
```

Comment:
This part of the diff just represents re-ordering the fields. Since the Region field changes based on what you put in for Country, I wanted to move Country up to the top.